controllers/user.js

```diff
@@ -162,12 +162,18 @@ async function getResponderCount(req, res) {
   if (user) {
     let responders = user.responders;
     let count = 0;
+    console.log(responders);
     for (let r of responders) {
       var responder = await UserModel.findOne({
         _id: new ObjectId(r.id)
       }).lean();
-      let availbilityStatus = await AvailbilityService.checkAvailabilityStatus(r.id);
-      if (availbilityStatus == true) count++;
+      try{
```

Comment:
i believe this is not needed (try catch block)

Reply:
it was something I had during debugging, I'll take it away

Reply:
> > Have you also been getting timeout errors? You can increase this timeout duration.
> > `Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (C:\Users\Nicholas\Documents\Code the Change\UHN-Backend-Server\tests\authentication.test.js)`
> 
> I haven't been getting time out issues, which test case is this?

The authentication ones.